### ISSUE TITLE
Some code128 barcodes are missing the last character

### ISSUE THREAD

**Issue by lkoba** [hudora/huBarcode#11]
This code produces the same bardcodes (the readable text changes, but the barcodes are the same). The first is missing the 9.

```
from hubarcode.code128 import Code128Encoder

encoder = Code128Encoder('123456789')
encoder.save('test1.png')

encoder = Code128Encoder('12345678')
encoder.save('test2.png')
```

**Comment by mdornseif** [comment]
Fascinating!

Can you provide a patch to fix that?

**Comment by mdornseif** [comment]
https://github.com/hudora/huBarcode/commit/7bdd6201517550f7d5057d51f04e08ea61e920ce should fix that. 

Published as huBarcode-0.63

**Comment by lkoba** [comment]
Thanks!
